recipes/bliss-automorphism/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "0.73.1" %}
+
+package:
+  name: bliss-automorphism
```

Comment:
there is a bliss package already

Reply:
Indeed, somehow I missed it.
One issue then is that SCIP relies on a patched version: https://github.com/ds4dm/Bliss/commit/faeb2d27854ab04efa8a512f9fc92ef546d1a12e
Should we vendor it inside the SCIP recipe? Or rename the recipe `bliss-automorphism` > `bliss-for-scip` ?
I am unsure if other recipes using `bliss` could accommodate the change.

Reply:
Can you send a PR to https://github.com/mkoeppe/bliss with your changes?

Reply:
Thanks, I did not know there was this repo. I will reach out to the owner, but given that our current patch is not built in a modular way, this could take time to get moving.

We are very determined to get the SCIP in conda as soon as possible. I believe it would make sense in the meantime to get our own Bliss with the SCIP source. Bliss is a very light package so this should not be a burden on the CI. What is you opinion on it?